app/upgrades/v5_0/upgrade.go

```diff
@@ -1,4 +1,4 @@
-package v5
+package v5_0 //nolint:revive,stylecheck
```

Comment:
Using an underscore in the package name throws a lint error but I think this should be an exception.